### ISSUE TITLE
Keyboard shortcut for foreign keys

### ISSUE THREAD

**Issue by mike-code** [TablePlus/TablePlus#1156]
I would find it very useful if navigating to foreign key record could be done using keyboard shortcut, however that would require possibility to move between record cells whilst now you can only select the whole row, correct?

img related

![image](https://user-images.githubusercontent.com/2964642/57054590-9ab14780-6ce9-11e9-862f-adf29ed433b5.png)

**Comment by huyphams** [comment]
Hi @mike-code you can navigate to foreign key by clicking the arrow, have you tried it?

**Comment by mike-code** [comment]
Correct. My suggestion here is for people (like myself) who try not to use mouse at all. Myself I almost never use mouse while working with datagrip and I learned that this gives me much faster control over the program so as an example I would probably like to be navigated to the related record rather than have all other entries filtered and I'd definitely need single cell selection (to edit or navigate to FK) to use mouse as little as needed. 

Still that's only a suggestion

**Comment by huyphams** [comment]
Understood, thanks.
